templates/ords-remix-jwt-sample/ords/modules/README.md

```diff
@@ -102,7 +102,7 @@ Here is a Diagram of how we will be structuring the API.
 
 Before we start modeling our endpoints we first need to understand who is going to use our API endpoints so we can take advantage of the ORDS Module feature to aggregate our API endpoints. For the sample app we have three possible users: 
 
-- **End User**: An End User should have limited access to our API endpoints and by limited access we mean READ ONLY access. 
+- **End User**: An End User should have limited access to our API endpoints and by limited access we mean READ ONLY access. End users can only access publicly available resources within our application. This means that all endpoints within the `euser` pattern are public and don't need authentication to be accessed.
```

Comment:
- In "our application", who it refers to? 🤔 should it be "the application" for clarity?
- I assume `euser` stands for `end user`, but it may not be that clear for final users.

Reply:
I changed it to "the application" as you suggested and added a parentheses with (the end user) after the mention of the `euser`.